sdk/communication/Azure.Communication.PhoneNumbers/tests/samples/Sample_PhoneNumbersClient.cs

```diff
@@ -32,20 +33,22 @@ public async Task PurchaseAndReleaseAsync()
 
             #region Snippet:SearchPhoneNumbersAsync
 
-            var capabilities = new PhoneNumberCapabilities(calling:PhoneNumberCapabilityType.None, sms:PhoneNumberCapabilityType.Outbound);
+            var capabilities = new PhoneNumberCapabilities(calling: PhoneNumberCapabilityType.None, sms: PhoneNumberCapabilityType.Outbound);
 
             var searchOperation = await client.StartSearchAvailablePhoneNumbersAsync(countryCode, PhoneNumberType.TollFree, PhoneNumberAssignmentType.Application, capabilities);
             //@@ await searchOperation.WaitForCompletionAsync();
-            /*@@*/ await WaitForCompletionAsync(searchOperation);
+            /*@@*/
+            await WaitForCompletionAsync(searchOperation);
```

Comment:
I believe this should be outside of the snippet region; otherwise, this appears in the generated samples in the readme file.